package.json

```diff
@@ -21,7 +21,7 @@
     "lint": "eslint --cache --ext .js ."
   },
   "peerDependencies": {
-    "webpack": "^4.0.0"
+    "webpack": "^5.0.0"
```

Comment:
Let's use >=4.0.0 here? In this case we won't drop webpack@4 support, but just add webpack@5.